### PR TITLE
Introducing UI config

### PR DESCRIPTION
**Pull request by fao89**
- galaxy_ng/settings.py to set STATIC_URL = "/static/"
- Add a new galaxy_ng view that 302 redirects / to /ui/
- proxy /ui* to /static/galaxy_ng/index.html

### CODE REVIEW

File: MANIFEST.IN [hunk 1]
```diff
@@ -0,0 +1 @@
+include galaxy_ng/app/webserver_snippets/*
```
Comment (newswangerd) [inline MANIFEST.IN:1]: @cutwater got rid of our manifest. If you're re adding it, please correct the name from `MANIFEST.IN` to `MANIFEST.in` we had a typo in our original filename.

Comment (newswangerd) [inline MANIFEST.IN:1]: rename `MANIFEST.IN` to `MANIFEST.in`

**Comment by bmbouter** [comment]
@fabricio-aguiar thank you for working on this. Can you share the contents of your main reverse proxy config so I could see what that looks like also?

**Comment by fabricio-aguiar** [comment]
> @fabricio-aguiar thank you for working on this. Can you share the contents of your main reverse proxy config so I could see what that looks like also?

```
(pulp) [vagrant@pulp3-source-fedora30 ansible-hub-ui]$ cat /etc/nginx/nginx.conf
# TODO: Support IPv6.
# TODO: Configure SSL certificates.
# TODO: Maybe serve multiple `location`s, not just one.

# The "nginx" package on fedora creates this user and group.
user nginx nginx;
# Gunicorn docs suggest this value.
worker_processes 1;
events {
    worker_connections 1024;  # increase if you have lots of clients
    accept_mutex off;  # set to 'on' if nginx worker_processes > 1
}

http {
    include mime.types;
    # fallback in case we can't determine a type
    default_type application/octet-stream;
    sendfile on;

    # If left at the default of 1024, nginx emits a warning about being unable
    # to build optimal hash types.
    types_hash_max_size 4096;

    upstream pulp-content {
         server 127.0.0.1:24816;
    }

    upstream pulp-api {
         server 127.0.0.1:24817;
    }

    server {
        # Gunicorn docs suggest the use of the "deferred" directive on Linux.
        listen 80 default_server deferred;
        server_name $hostname;

        # The default client_max_body_size is 1m. Clients uploading
        # files larger than this will need to chunk said files.

        # Gunicorn docs suggest this value.
        keepalive_timeout 5;

        # Path to Pulp's static files.
        root /var/lib/pulp/assets/;

        location /assets/ {
            # static files are served through whitenoise - http://whitenoise.evans.io/en/stable/
            proxy_redirect off;
            proxy_pass http://127.0.0.1:24817/assets/;
        }

        location /pulp/content/ {
            proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
            proxy_set_header X-Forwarded-Proto $scheme;
            proxy_set_header Host $http_host;
            # we don't want nginx trying to do something clever with
            # redirects, we set the Host: header above already.
            proxy_redirect off;
            proxy_pass http://127.0.0.1:24816;
        }

        location /pulp/api/v3/ {
            proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
            proxy_set_header X-Forwarded-Proto $scheme;
            proxy_set_header Host $http_host;
            # we don't want nginx trying to do something clever with
            # redirects, we set the Host: header above already.
            proxy_redirect off;
            proxy_pass http://127.0.0.1:24817;
        }

        location /auth/login/ {
            proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
            proxy_set_header X-Forwarded-Proto $scheme;
            proxy_set_header Host $http_host;
            # we don't want nginx trying to do something clever with
            # redirects, we set the Host: header above already.
            proxy_redirect off;
            proxy_pass http://127.0.0.1:24817;
        }

        include conf.d/*;

        location / {
            # checks for static file, if not found proxy to app
            try_files $uri @proxy_to_api;
        }

        location @proxy_to_api {
            proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
            proxy_set_header X-Forwarded-Proto $scheme;
            proxy_set_header Host $http_host;
            # we don't want nginx trying to do something clever with
            # redirects, we set the Host: header above already.
            proxy_redirect off;
            proxy_pass http://pulp-api;

            proxy_next_upstream http_404;  # Try the other service upon 404
        }
    }
}
```

File: galaxy_ng/app/webserver_snippets/apache.conf [hunk 2]
```diff
@@ -0,0 +1,6 @@
+RewriteEngine  on
+RewriteCond "%{REQUEST_FILENAME}"       !-f
+RewriteCond "%{REQUEST_FILENAME}"       !-d
+RewriteRule "^/ui*" "http://${pulp-api}/static/galaxy_ng/index.html" [P]
+ProxyPass "/ui/" "http://${pulp-api}/static/galaxy_ng/index.html"
+ProxyPassReverse "/ui/" "http://${pulp-api}/static/galaxy_ng/index.html"
```
Comment (fabricio-aguiar) [inline galaxy_ng/app/webserver_snippets/apache.conf:6]: https://httpd.apache.org/docs/2.4/rewrite/proxy.html

File: MANIFEST.in [hunk 3]
```diff
@@ -0,0 +1 @@
+include galaxy_ng/app/webserver_snippets/*
```
Comment (bmbouter) [inline MANIFEST.in:1]: nitpick: s/include/Include/

File: MANIFEST.in [hunk 4]
```diff
@@ -0,0 +1 @@
+Include galaxy_ng/app/webserver_snippets/*
```
Comment (newswangerd) [inline MANIFEST.in:1]: `webserver_snippets` isn't getting included in the wheel. Might need to change `Include` -> `include`

**Pull request merged**